libwild/src/args.rs

```diff
@@ -194,8 +194,10 @@ pub(crate) struct Input {
 
 #[derive(Debug, Eq, PartialEq)]
 pub(crate) enum InputSpec {
+    /// Path (possibly just a filename) to the file.
     File(Box<Path>),
-    Lib(Box<str>),
+    /// Name of the library and whether it already has suffix and prefix.
+    Lib(Box<str>, bool),
```

Comment:
I wonder how it might look if instead of adding a bool to this variant, we had a separate variant. e.g. `Search(Box<str>)` which indicates to search for a file with the indicated name.

Reply:
Sure, I'll prepare it.